js/src/functions/httpTriggerBadAsync.js

```diff
@@ -0,0 +1,21 @@
+// NOT RECOMMENDED PATTERN

```

Comment:
Nit: rather than say "Not Recommended", perhaps we should be even more forceful and say something like "Do not use this code" or "This does not work".

"Not recommended" has the connotation that you can use it, but the experience won't be great. But, in this case, it's more that you are saying the code is to be avoided at all costs.